src/views/virtualmachines/details/tabs/configuration/network/components/list/NetworkInterfaceRow.tsx

```diff
@@ -17,36 +24,45 @@ export type NetworkInterfaceRowProps = {
 const NetworkInterfaceRow: FC<
   RowProps<
     NetworkPresentation,
-    { isPending: (network: V1Network) => boolean; vm: V1VirtualMachine }
+    {
+      isInterfaceEphemeral: (
+        network: V1Network,
+        iface: V1VirtualMachineInstanceNetworkInterface,
+      ) => undefined | V1VirtualMachineInstanceNetworkInterface;
+      isPending: (network: V1Network) => boolean;
+      vm: V1VirtualMachine;
+    }
```

Comment:
Why passing this function in the RowProps? I think you changed idea in the middle of the work don't you?

Reply:
just to keep the schema , to be same as isPending